frontend/home/homeController.js

```diff
@@ -35,6 +36,18 @@
             $state.go('app.institution.timeline', {institutionKey: institutionKey});
         };
 
+        homeCtrl.showEvent = function showEvent(event) {
```

Comment:
showEvent seams to be an action that acts directly to render the event in the view. But, this is just a function that verifies if the event already happened.